### DISCUSSION TITLE
Configuring UI Date picker modal [Aug 07]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6180] [Discord Threads]
> 🙋‍♂️ **funapple**  
## is there a way to globally set first day of week as monday in date selection?  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1123881325301071882/f1e44c0015438911.png)

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Not supported  
#  
> 🙋‍♂️ **funapple**  
#### bruh...   
#### but will it be supported? half of the world lives in world where monday is first day of week  
#  
> 🙋‍♂️ **.berteh**  
#### +1 for some config there. I'd like to have Monday as first day, and display the week number as well.  
#  
> 🙋‍♂️ **.berteh**  
#### Any place I could start to have a look to hack/modify the current code?  
#  
> 👨‍🦲 **dstala**  
#### will discuss with engineering team & update  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1123881325523390484)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
